eclass/perl-module.eclass

```diff
@@ -120,11 +92,11 @@ case ${EAPI} in
 		esac
 
 		if [[ "${GENTOO_DEPEND_ON_PERL_SUBSLOT:-yes}" != "yes" ]]; then
```

Comment:
@akhuettel don't we just want to remove these directly? nothing uses it anymore, for the stuff that does, it just a dead var?

Reply:
I'm torn on this. I'm kinda tempted to just have a list of them where we die if any are defined, because I don't want things to silently use the wrong vars for years. We already saw that with DO_TEST or whatever it was

Reply:
Here's an idea: define an array with all of the banned variables in (dereferenced). If the array is non-empty, die, "deprecated variables found. Please see eclassdocs."

It'll take up very little space?

Reply:
Was just a suggestion, let's keep it like it is right now